modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,50 @@
+# Imaging Browser
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data-entry personnel
+2. Site coordinators and data-monitors validating the data
+
+## Scope
+
+The Candidate Parameters modules stores all information 
+for a candidate that are not timepoint dependent. All data 
+being collect during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions. Any one of them
```

Comment:
Either one, there's only two. You can probably edit out the whole first sentence too and just say something like "Either of the following two permissions is sufficient to access the candidate parameter's module:"

Reply:
just staying consistent with the imaging browser sentence.
`The imaging browser module uses the following permissions. Any one of them (except imaging_browser_qc) is sufficient to have access to the module.`

Reply:
i agree with dave's comment